<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Damage_Group</class>
 <widget class="QDialog" name="Damage_Group">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Damage Group</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Condition Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Condition</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Conditon Value</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="condition_type_combo">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="condition_combo">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="condition_value_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="condition_table">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Condition Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Condition</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Condition Value</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="name_line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="damage_group_add_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_condition_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add </string>
   </property>
  </widget>
  <widget class="QComboBox" name="element_combo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PIPE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PUMP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NODE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TANK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RESERVOIR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GENERAL NODE</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="damage_group_table">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>231</width>
     <height>291</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Element Type</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="damage_group_remove_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="damage_group_edit_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_data_table">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>100</y>
     <width>101</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="move_list_value_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>220</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;----------</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_file_button">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>List By File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_condition_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Damage_Group</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Damage_Group</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
